kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNRuntimeTest.java

```diff
@@ -1112,9 +1114,26 @@ public void testUnknownVariable2() {
         assertThat( dmnModel, notNullValue() );
         assertThat( formatMessages( dmnModel.getMessages() ), dmnModel.getMessages().size(), is( 1 ) );
         assertThat( dmnModel.getMessages().get( 0 ).getMessageType(), is( DMNMessageType.ERR_COMPILING_FEEL ) );
-        assertThat( dmnModel.getMessages().get( 0 ).getMessage(), containsString( "Unknown variable 'liquidAssetsAmt'" ) );
+        assertThat( dmnModel.getMessages().get( 0 ).getMessage(), containsString( "Unknown variable 'Borrower.liquidAssetsAmt'" ) );
```

Comment:
according to implementation currently on master, the error message should return the FQN.